frontend/src/views/Game.tsx

```diff
@@ -127,20 +176,23 @@ function GamePage() {
   return (
     <FlexContainer>
       <FlexInfoBar>
-        Room:
-        {' '}
-        {roomId || 'An unknown room'}
-        {` (${game?.room?.users?.length} players)`}
-      </FlexInfoBar>
-      <FlexInfoBar>
-        You are
-        {' '}
-        {currentUser != null ? currentUser.nickname : 'An unknown user'}
-      </FlexInfoBar>
-      <FlexInfoBar>
-        <GameTimerContainer gameTimer={game ? game.gameTimer : null} />
+        <FlexLeft>
+          Room:
+          {' '}
+          {roomId || 'N/A'}
+        </FlexLeft>
+        <FlexCenter>
+          <GameTimerContainer gameTimer={gameTimer || null} />
+        </FlexCenter>
+        <FlexRight>
+          <TextLink to="/">Exit Game</TextLink>
```

Comment:
Good idea - implemented this with a confirmation popup. One thing to note: the logic I wrote is only on the frontend, so on the backend they'd still be a part of the game. So like if someone submitted and got first place but decided to leave before the game officially ends, they'd still be on the leaderboard instead of removed. Personally I'm okay with that design choice, but we can also create a card in the backlog if we think they should also be removed on the backend too

Reply:
I think that choice makes sense -- we can revisit that in the future if it becomes a problem, but sounds good for now.

Reply:
I think it might make sense to disconnect at least -- still keeping the player's history and whatnot in the game, but making them inactive. This means that if they try to create a new room, they will connect to that instead; right now, this doesn't work as the user is still connected to the other room.

Reply:
👍